src/rdkafka_conf.c

```diff
@@ -1890,6 +1899,32 @@ void rd_kafka_conf_set_open_cb (rd_kafka_conf_t *conf,
 }
 #endif
 
+rd_kafka_conf_res_t
+rd_kafka_conf_set_ssl_cert_verify_cb(rd_kafka_conf_t *conf,
+    int (*ssl_cert_verify_cb) (char *cert, size_t len, void *opaque)) {
+#ifdef __mips__
+    return RD_KAFKA_RESP_ERR__NOT_IMPLEMENTED;
+#endif
+#if WITH_SSL
+    conf->ssl.cert_verify_cb = ssl_cert_verify_cb;
+#else
+    RD_UNUSED(conf);
+    RD_UNUSED(ssl_cert_verify_cb);
+#endif
+
+    return RD_KAFKA_RESP_ERR_NO_ERROR;
+}
+
+void rd_kafka_conf_set_ssl_cert_retrieve_cb(rd_kafka_conf_t *conf,
```

Comment:
add rd_kafka_conf_res_t return type and return UNKNOWN if !WITH_SSL.

also remove UNUSED here